client/src/CategoryForm.js

```diff
@@ -55,20 +58,33 @@ function CategoryForm() {
               <label className="form-label" htmlFor="NavBackImg">
                 NavBackImg
               </label>
-              <input type="text" className="form-control" id="NavBackImg" name="NavBackImg" onChange={onChange} value={data.NavBackImg} />
+              <PhotoInput
+                className="card"
+                id="NavBackImg"
+                name="NavBackImg"
+                value={data.NavBackImg}
+                valueUrl={data.NavBackImgUrl}
+                onChange={onChange}>
+                <div className="card-body">
+                  <div className="card-text">Drag-and-drop a photo file here, or click here to browse and select a file.</div>
+                </div>
+              </PhotoInput>
```

Comment:
Here we see how the PhotoInput component can be used in a form to allow the user to attach image files.